pkg/lib/cockpit-components-table.jsx

```diff
@@ -170,7 +170,8 @@ export class ListingTable extends React.Component {
                     parent: rowIndex - 1,
                     cells: [{ title: currentValue.expandedContent }],
                     fullWidth: true, noPadding: true,
-                    rowId: currentValue.rowId ? cockpit.format("$0-expanded", currentValue.rowId) : undefined
+                    rowId: currentValue.rowId ? cockpit.format("$0-expanded", currentValue.rowId) : undefined,
+                    props: { key: cockpit.format("$0-expanded", currentValue.props.key) },
```

Comment:
It's a bit expensive to use cockpit.format for that -- a simple `+` would certainly do. :-)